201-alert-to-queue-with-logic-app/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+       "EndpointName": {
+            "value": "myendpoint"
+        },
+        "QueueName": {
+            "value": "myqueue"
+        },
+        "StorageAccountName": {
```

Comment:
We're going to enable automated testing soon that will generate a random string for things such as storage account. To enable that can you replace the value below with 5 '#'s, "#####"